src/reducers/search.js

```diff
@@ -89,6 +90,11 @@ export default function search(state = initialState, action) {
         searchResultsPageUrl: action.url
       });
 
+    case SEARCH_RESULTS_MIN_LENGTH_REQUIRED:
```

Comment:
The reducer is missing the initial state for _minLengthRequired_. 

(Same on the autocomplete reducer, but as per other comment maybe remove from there completely)